lib/hexpm_web/router.ex

```diff
@@ -62,6 +62,9 @@ defmodule HexpmWeb.Router do
     post "/login", LoginController, :create
     post "/logout", LoginController, :delete
 
+    get "/login/web_auth", WebAuthController, :show
+    post "/login/web_auth/submit", WebAuthController, :submit
```

Comment:
I would rather move this to its own controller so we don't have a single controller handling both browser and API requests.

Reply:
I don't this was done.